pages/source/documentation/index.md

```diff
@@ -688,8 +737,8 @@ for more details.
 Each **Action** phase can be in one of the following states:
 
 -   **Pending** - Not yet executed.
--   **Running** - Action phase is being executed right now.
--   **Success** - Action phase execution finished successfully.
+-   **Running** - Action phase ?which? is being executed right now.
+-   **Success** - Action phase ?which? execution finished successfully.
```

Comment:
the ?which? was actually my question, not suggestion :-) I think we can remove these now